jina/flow/base.py

```diff
@@ -64,6 +61,9 @@ def __init__(self, args: Optional['argparse.Namespace'] = None, env: Optional[Di
         will generate 4 Peas, but it will force the :class:`GatewayPea` to take BIND role,
         as the head and tail routers are removed.
 
+        :param kwargs: other keyword arguments that will be shared by all pods in this flow
```

Comment:
- **P**ods, **F**low
- Should we capitalize first letter (i.e. **O**ther)?

Reply:
Transformed to uppercase most of them (in this folder)

Reply:
> Should we capitalize first letter (i.e. Other)?

I don't think we're doing this atm. Also the linter doesn't complain about it, so I wouldn't care about it too much